CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
this file should be removed

Reply:
Why? It's a standard file that's in many repositories (actually I copied it from the Archive Repository).

Reply:
Because the idea of CoC is stupid. I don't understand the fashion of the recent years to add it.

Reply:
Fine, removed.